src/markdown/tutorial/part-1/04-component-basics.md

```diff
@@ -156,6 +156,11 @@ ember generate component-test jumbo
 ```
 
 ```run:command hidden=true cwd=super-rentals
+# hack until https://github.com/emberjs/ember.js/pull/20988 is fixed
```

Comment:
While this will make the tutorial build, anybody following it will see an error and be blocked